src/App.tsx

```diff
@@ -1,56 +1,102 @@
+    setSelectPerson(selectedPerson);
+    setInputValue(selectedPerson.name);
+    setFocusInput(false);
+  }, []);
+
+  const filteredPerson = people.filter((person) => {
+    return person.name.toLowerCase().includes(inputValue.toLowerCase());
+  });
+  const title = selectPerson ? `${selectPerson.name} (${selectPerson.born} - ${selectPerson.died})` : 'No matching suggestions';
+
+  const handleDelete = () => {
+    setInputValue('');
+    setSelectPerson(null);
+  };
 
   return (
     <main className="section">
       <h1 className="title">
-        {`${name} (${born} = ${died})`}
+        {!inputValue ? 'Сhoose a name' : title}
       </h1>
 
       <div className="dropdown is-active">
         <div className="dropdown-trigger">
           <input
+            onChange={handleChange}
+            value={inputValue}
             type="text"
             placeholder="Enter a part of the name"
             className="input"
+            onFocus={() => setFocusInput(true)}
           />
+          {
+            inputValue
+            && (
+              <button
+                type="button"
+                className="delete"
+                onClick={handleDelete}
+                aria-label="Clear input"
+              />
+            )
+          }
         </div>
 
-        <div className="dropdown-menu" role="menu">
-          <div className="dropdown-content">
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Bernard Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Antone Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter de Decker</p>
+        {focusInput
+          && (
+            <div className="dropdown-menu" role="menu">
+              <div className="dropdown-content">
+                <div className="dropdown-item">
+                  {inputValue ? (
+                    filteredPerson.map((person) => (
+                      // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events
+                      <p
+                        key={person.name}
+                        className="has-text-link"
+                        onClick={() => handlePersonClick(person)}
+                      >
+                        {person.name}
+                      </p>
+                    ))
+                  ) : (
+                    peopleFromServer.map((person) => (
+                      // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events
```

Comment:
It's not a good practice to disable eslint rules unless it's really necessary. You should add 'onKeyPress' event and make your element interactive.